File: lab1-project2_1_Ладик/lab1Task4/app/src/main/java/com/example/lab1task4/MainActivity.java
package com.example.lab1task4;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button but = findViewById(R.id.button);
        TextView tg = findViewById(R.id.tg);
        but.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                tg.setVisibility(View.VISIBLE);
            }
        });
    }
}

File: lab1-project2_2_Ladik/app/src/main/java/com/example/lab1task42/MainActivity.kt
package com.example.lab1task42

import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val but = findViewById<Button>(R.id.button)
        val tg = findViewById<TextView>(R.id.tg)

        but.setOnClickListener{
            tg.visibility = View.VISIBLE
        }
    }
}